cognite_toolkit/_cdf_tk/loaders/_base_loaders.py

```diff
@@ -168,7 +168,9 @@ def dump_id(cls, id: T_ID) -> dict[str, Any]:
 
     @classmethod
     @abstractmethod
-    def get_required_capability(cls, items: T_CogniteResourceList | None) -> Capability | list[Capability]:
+    def get_required_capability(
+        cls, items: T_CogniteResourceList | None, read_only: bool
```

Comment:
Not sure if it is worth it, but would expect a default value here
```suggestion
        cls, items: T_CogniteResourceList | None, read_only: bool = False
```